src/ophyd_async/core/_detector.py

```diff
@@ -87,7 +93,7 @@ def get_deadtime(self, exposure: float | None) -> float:
         """For a given exposure, how long should the time between exposures be"""
 
     @abstractmethod
-    async def prepare(self, trigger_info: TriggerInfo):
+    async def prepare(self, trigger_info: TriggerInfo) -> Any:
```

Comment:
Is the return value of this always going to be the same for a given DetectorController, or is it going to differ depending on the type of experiment? DetectorController could be generic on the return value of `prepare`, so we have some type checking?

Reply:
I don't think this should ever return anything...
@jwlodek are there examples where it does return something?

Reply:
No, I don't think so. For some reason vscode built in type checking was reporting that it was returning a coro and such having a type mismatch. I think it should just be None though. I will switch it back